### PR TITLE
Added street view to geography action

### PR DESCRIPTION
**Pull request by sarahc7**
Added street view and updated UI.

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -66,46 +66,3 @@ header {
 .next-question {
     margin-top: 42px;
 }
-
-.hide-with-space {
```
Comment (gaurnett) [inline public/css/main.css]: Did you delete the .hide-with-space and .pic-table?

File: public/index.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
         <!-- Load custom CSS -->
         <link rel="stylesheet" href="css/main.css" />
-        <link rel="stylesheet" href="css/reading.css" />
```
Comment (gaurnett) [inline public/index.html:52]: I think you should be adding a new link tag and not remove the reading.css files

**Review by gaurnett — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 2 above)
Comment (Arboghast) [reply public/index.html:52]: +1

File: public/js/geography/geography-question.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        this.front = this.createCard("front");
+        this.back = this.createCard("back");
+    }
+
+    /**
+     * Flips card to reveal answer.
+     */
+    flipCard() {
+        this.cardInner.classList.toggle("flip-card-inner");
```
Comment (Arboghast) [inline public/js/geography/geography-question.js:22]: Nice, didn't know this function existed!

**Review by Arboghast — APPROVED** [review]
LGTM, just re-add the reading.css link tag and you're good.

File: public/css/main.css (hunk 1 above)
Comment (sarahc7) [reply public/css/main.css]: Yeah I accidentally did - thanks for catching that!

File: public/index.html (hunk 2 above)
Comment (sarahc7) [reply public/index.html:52]: Thanks for catching this too

**Review by gaurnett — APPROVED** [review]
LGTM

**Pull request merged**